recipes/libxml2/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ `uname` == 'Linux' ]];
+then
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+./autogen.sh
+./configure --prefix="${PREFIX}" \
+            --with-iconv="${PREFIX}" \
+            --with-icu \
+            --with-lzma="${PREFIX}" \
+            --with-python="${PREFIX}" \
+            --with-zlib="${PREFIX}"
+make
+# Correct weirdly linked library paths.
+if [[ `uname` == 'Darwin' ]];
+then
+    LIBDIR="${PREFIX}/lib"
+    LIBICUDATA="`cd ${LIBDIR} && ls libicudata.*.*.dylib`"
+    find "${SRC_DIR}/.libs" -type f | xargs -I {} install_name_tool -change "../lib/${LIBICUDATA}" "${PREFIX}/lib/${LIBICUDATA}" {} || true
+    find "${SRC_DIR}/python/.libs" -type f | xargs -I {} install_name_tool -change "../lib/${LIBICUDATA}" "${PREFIX}/lib/${LIBICUDATA}" {} || true
```

Comment:
Since we are removing Python support we won't need this last `find` line. Instead of deleting it, could we just comment it so that we remember how to do this in the future if we ever do get it to work.